### PR TITLE
adopt an issue labelling scheme.

### PR DESCRIPTION
**Pull request by raulk**
Inspired by https://github.com/libp2p/libp2p/blob/master/ISSUE-LABELLING.md.

We can use as much of this as we want. For now, I'm interested in the area, hints, statuses, and priority labels.

### CODE REVIEW

**Comment by Robmat05** [comment]
LGTM

File: .github/labels.yml [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+- name: area/runners
+  color: 0bb1ed
+  description: "Area: Runners"
+- name: area/builders
+  color: 0bb1ed
+  description: "Area: Builders"
+- name: area/infrastructure
+  color: 0bb1ed
+  description: "Area: Infrastructure"
+- name: area/infra
```
Comment (nonsense) [inline .github/labels.yml]: This seems off, should be `area/specs`

File: .github/labels.yml [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+  color: 0bb1ed
+  description: "Area: Specs"
+- name: area/specs
```
Comment (nonsense) [inline .github/labels.yml]: `area/docs`

File: .github/labels.yml [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+  color: 0bb1ed
+  description: "Area: Documentation"
+- name: area/docs
```
Comment (nonsense) [inline .github/labels.yml]: `area/testing`

**Pull request merged**